data-prepper-plugins/common/src/main/java/com/amazon/dataprepper/plugins/buffer/BlockingBuffer.java

```diff
@@ -80,15 +92,29 @@ public BlockingBuffer(final String pipelineName) {
 
     @Override
     public void doWrite(T record, int timeoutInMillis) throws TimeoutException {
+        long nanos = TimeUnit.MILLISECONDS.toNanos(timeoutInMillis);
+        final int c;
```

Comment:
nit: why declare c here? if you're not using it in the catch/finally block, it can be declared inline with the initialization